modules/covid_vaccine/app/jobs/covid_vaccine/confirmation_email_job.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+require 'va_notify/service'
+
+module CovidVaccine
+  class ConfirmationEmailJob
+    include Sidekiq::Worker
+    include SentryLogging
+    sidekiq_options expires_in: 1.day, retry: 2
+
+    STATSD_ERROR_NAME = 'worker.covid_vaccine_confirmation_email.error'
+    STATSD_SUCCESS_NAME = 'worker.covid_vaccine_confirmation_email.success'
+
+    def perform(email, date, sid)
```

Comment:
Based on the proposed flow: https://github.com/department-of-veterans-affairs/va.gov-team/issues/22118

there will never be a `sid` at this point in the process because we will not have submitted to vetext at this point in the process. The id that should be used to trigger this job is the record_id, like here:
https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/covid_vaccine/app/jobs/covid_vaccine/submission_job.rb#L10

Reply:
(i.e just the standard activerecord `id` field for the RegistrationSubmission object

Reply:
oh, for some reason looking at the schema I thought we had replaced id with sid and we were just going to generate something randomish.